src/pages/MemoriesPage.js

```diff
@@ -37,30 +41,68 @@ export default function MemoriesPage({
         </BackToCam>
         <h2>Your memories</h2>
       </Title>
+      <FormSearch onSubmit={handleSearch}>
+        <input
+          type="text"
+          onChange={event => setSearchMemory(event.target.value.toLowerCase())}
+          placeholder="Search your memory"
+          name="search"
+          autoComplete="off"
+          aria-label="Search your memory"
+        />{' '}
+        <SearchButton>
+          <SearchIcon src={search} alt="" />
+        </SearchButton>
+      </FormSearch>
       <ListWrapper>
-        {memories.map(({ image, title, date, text, id }) => (
-          <ListItem key={id}>
-            <IconButton onClick={() => onDelete(id)}>
-              <Icon src={DeleteIcon} alt="" />
-            </IconButton>
-            <MemoryItem
-              image={image}
-              title={title}
-              date={date}
-              text={text}
-              id={id}
-              onDetail={() => onDetail(image, title, date, text, id)}
-            />
-          </ListItem>
-        ))}
+        {searchMemory
+          ? memories
+              .filter(memory =>
+                memory.title.toLowerCase().includes(searchMemory)
+              )
+              .map(({ image, title, date, text, id }) => (
+                <ListItem key={id}>
+                  <IconButton onClick={() => onDelete(id)}>
+                    <DeletedIcon src={DeleteIcon} alt="" />
+                  </IconButton>
+                  <MemoryItem
+                    image={image}
+                    title={title}
+                    date={date}
+                    text={text}
+                    id={id}
+                    onDetail={() => onDetail(image, title, date, text, id)}
+                  />
+                </ListItem>
+              ))
+          : memories.map(({ image, title, date, text, id }) => (
+              <ListItem key={id}>
+                <IconButton onClick={() => onDelete(id)}>
+                  <DeletedIcon src={DeleteIcon} alt="" />
+                </IconButton>
+                <MemoryItem
+                  image={image}
+                  title={title}
+                  date={date}
+                  text={text}
+                  id={id}
+                  onDetail={() => onDetail(image, title, date, text, id)}
+                />
+              </ListItem>
+            ))}
```

Comment:
nice! maybe you could add a little hint if there is no result for the search :)

Reply:
Oh, good point!